src/server/pps/server/api_server.go

```diff
@@ -1954,8 +1967,18 @@ func RepoNameToEnvString(repoName string) string {
 	return strings.ToUpper(repoName)
 }
 
-// Convert a persist.JobInfo into a Kubernetes batch.Job spec
-func job(kubeClient *kube.Client, jobInfo *persist.JobInfo, jobShimImage string, jobImagePullPolicy string) (*batch.Job, error) {
+type jobOptions struct {
+	labels             map[string]string
+	parallelism        int32
+	userImage          string
+	jobShimImage       string
+	jobImagePullPolicy string
+	jobEnv             []api.EnvVar
+	volumes            []api.Volume
+	volumeMounts       []api.VolumeMount
+}
+
+func getJobOptions(kubeClient *kube.Client, jobInfo *persist.JobInfo, jobShimImage string, jobImagePullPolicy string) (*jobOptions, error) {
```

Comment:
I think I had it that way, but then it conflicts w the `jobOptions` struct I define a few lines above. In this case what's the best convention?